cudl-staging/terraform.tfvars

```diff
@@ -99,117 +109,215 @@ transform-lambda-bucket-sqs-notifications = [
-    "jar_path"    = "release/uk/ac/cam/lib/cudl/awslambda/AWSLambda_Data_Transform/0.16/AWSLambda_Data_Transform-0.16-jar-with-dependencies.jar"
-    "queue_name"  = "CUDLPackageDataUIQueue"
-    "timeout"     = 900
-    "memory"      = 512
-    "handler"     = "uk.ac.cam.lib.cudl.awslambda.handlers.UIFileDBHandler::handleRequest"
-    "runtime"     = "java11"
+    "name"                  = "AWSLambda_CUDLPackageData_HTML_to_HTML_Translate_URLS"
+    "description"           = "Processes HTML files from source data format into the releases data format by transforming the URL paths"
+    "jar_path"              = "release/uk/ac/cam/lib/cudl/awslambda/AWSLambda_Data_Transform/1.0/AWSLambda_Data_Transform-1.0-jar-with-dependencies.jar"
+    "queue_name"            = "CUDLPackageDataQueue_HTML"
+    "subnet_names"          = ["cudl-subnet-private1-eu-west-1a"]
+    "use_datadog_variables" = false
+    "timeout"               = 900
+    "memory"                = 512
+    "handler"               = "uk.ac.cam.lib.cudl.awslambda.handlers.ConvertHTMLIdsHandler::handleRequest"
+    "runtime"               = "java11"
+  },
+  {
+    "name"                  = "AWSLambda_CUDLPackageData_FILE_UNCHANGED_COPY"
+    "description"           = "Copies file from the source s3 bucket into the destination (release) s3 bucket, unchanged"
+    "jar_path"              = "release/uk/ac/cam/lib/cudl/awslambda/AWSLambda_Data_Transform/1.0/AWSLambda_Data_Transform-1.0-jar-with-dependencies.jar"
+    "queue_name"            = "CUDLPackageDataQueue_FILES_UNCHANGED_COPY"
+    "subnet_names"          = ["cudl-subnet-private1-eu-west-1a"]
+    "use_datadog_variables" = false
+    "timeout"               = 900
+    "memory"                = 512
+    "other_filters"         = "cudl.dl-dataset.json|cudl.ui.json"
+    "handler"               = "uk.ac.cam.lib.cudl.awslambda.handlers.CopyFileHandler::handleRequest"
+    "runtime"               = "java11"
+  },
+  {
+    "name"                  = "AWSLambda_CUDLPackageData_JSON_to_JSON_Translate_URLS"
+    "description"           = "Transforms the collection json file into a json format with suitable paths for the viewer / db"
+    "jar_path"              = "release/uk/ac/cam/lib/cudl/awslambda/AWSLambda_Data_Transform/1.0/AWSLambda_Data_Transform-1.0-jar-with-dependencies.jar"
+    "queue_name"            = "CUDLPackageDataQueue_Collections"
+    "subnet_names"          = ["cudl-subnet-private1-eu-west-1a"]
+    "use_datadog_variables" = false
+    "timeout"               = 900
+    "memory"                = 512
+    "handler"               = "uk.ac.cam.lib.cudl.awslambda.handlers.ConvertJSONIdsHandler::handleRequest"
+    "runtime"               = "java11"
+  },
+  {
+    "name"                     = "AWSLambda_CUDLPackageData_TEI_Processing"
+    "image_uri"                = "563181399728.dkr.ecr.eu-west-1.amazonaws.com/cudl-tei-processing@sha256:a35d1beea05ffffe1461f260648d1bcf6521ceba8309cd9d071a34422a288015"
```

Comment:
The `image_uri` here will need to exist in the main AWS account

Reply:
Since the configuration for the production, staging and dev will not be backwards compatible, it cannot be deployed until the whole new architecture is in place.  For this reason I have added the required parameters as placeholders to dev, staging and live until we are able to put in the true values.